src/foam/u2/ButtonSize.js

```diff
@@ -11,8 +11,8 @@ foam.ENUM({
   properties: ['iconSize'],
 
   values: [
-    { name: 'LARGE', label: 'Large', iconSize: 32 },
-    { name: 'MEDIUM', label: 'Medium', iconSize: 24 },
-    { name: 'SMALL', label: 'Small', iconSize: 16 }
+    { name: 'LARGE', label: 'Large', iconSize: '2em' },
+    { name: 'MEDIUM', label: 'Medium', iconSize: '1.5em' },
+    { name: 'SMALL', label: 'Small', iconSize: '1em' }
```

Comment:
Could we have these be the same sizes as now? 2.25, 1.71, 1.15 respectively.

Reply:
yup, updated thanks!